src/main/scala/tech/cryptonomic/conseil/michelson/JsonToMichelson.scala

```diff
@@ -0,0 +1,15 @@
+package tech.cryptonomic.conseil.michelson
+
+import tech.cryptonomic.conseil.michelson.parser.JsonParser
+import tech.cryptonomic.conseil.michelson.presenter.MichelsonPresenter._
+
+class JsonToMichelson {
```

Comment:
Could we leave ScalaDoc comments for all classes and public functions? The descriptions could be very short. For types corresponding to the JSON format or Michelson format, some examples in the comments would be an added bonus.

Reply:
I don't like commenting classes and methods with pretty much the same words as their names. I did it anyway to be consistent. On the other hand, I find code examples very useful :).